qx-lock.json

```diff
@@ -0,0 +1,24 @@
+{
+  "libraries": [
+    {
+      "library_name": "Qooxdoo API Viewer",
+      "library_version": "0.1.16",
+      "path": "qx_packages/qooxdoo_qxl_apiviewer_v0_1_16",
+      "uri": "qooxdoo/qxl.apiviewer",
+      "repo_name": "qooxdoo/qxl.apiviewer",
+      "repo_tag": "v0.1.16"
+    },
+    {
+      "library_name": "versionlabel",
+      "library_version": "0.1.2",
+      "path": "qx_packages/qooxdoo_qxl_versionlabel_v0_1_2",
+      "uri": "qooxdoo/qxl.versionlabel",
+      "repo_name": "qooxdoo/qxl.versionlabel",
+      "repo_tag": "v0.1.2"
+    }
+  ],
+  "version": "2.1",
+  "requires": {
+    "qooxdoo/qxl.apiviewer": "^0.1.16"
```

Comment:
This is a bug from before my last fixes. This should go into Manifest.json and should be removed here.